fastlane/swift/ConfigItem.swift

```diff
@@ -0,0 +1,129 @@
+// ConfigItem.swift
+// Copyright (c) 2021 FastlaneTools
+
+//
+//  ** NOTE **
+//  This file is provided by fastlane and WILL be overwritten in future updates
+//  If you want to add extra functionality to this project, create a new file in a
+//  new group so that it won't be marked for upgrade
+//
+
+import Foundation
+
+public enum ConfigItem<T> {
```

Comment:
A small nit pick maybe but thoughts on renaming this to `ConfigValue`? 😇 A config item is something a little more descriptive on the Ruby side where this is specifically just the value 🤷‍♂️

Reply:
I agree with renaming. This is a bit confusing to me. Since it handles some sort of optionality on Swift side, `OptionalConfigValue` might make sense slightly more? 😆  
https://github.com/fastlane/fastlane/pull/18544#discussion_r625198720

Reply:
I ended up renaming as @ainame suggested if that's fine with y'all 😄